File: JS/canVote-3.js
function canVote(age) {
    if (age == null) return "Oops!, age is not passed"
    if (age < 18) return "Cannot vote, Sorry!"
    return "You can vote!"
}

console.log(canVote(null));
console.log(canVote(17));
console.log(canVote(18));
console.log(canVote(28));
console.log(canVote(10));

File: Python/canVote-3.py
def can_vote(age):
    if age is None:
        return "Oops!, age is not passed"
    if age < 18:
        return "Cannot vote, Sorry!"
    return "You can vote!"

print(can_vote(None))
print(can_vote(17))
print(can_vote(18))
print(can_vote(28))
print(can_vote(10))
